### PR TITLE
Each e-mail sent out contains a Visit ID at the bottom of each email

### PR DESCRIPTION
**Pull request by jasiek**

### CODE REVIEW

File: spec/lib/metrics_logger_spec.rb [hunk 1]
```diff
@@ -14,13 +10,13 @@
     Timecop.return
   end
 
-  it "logs a visit request" do
+  it "logs a isit request" do
```
Comment (SteveMarshall) [inline spec/lib/metrics_logger_spec.rb]: Typo!

**Comment by SteveMarshall** [comment]
Looks good to go to me, unless @matharden wants anything more done with visit IDs in the emails?

File: app/views/mailers_common/_booking_confirmation.html.haml [hunk 2]
```diff
@@ -99,3 +99,7 @@
 %h1{ style: 'margin:1em 0;font: bold 24px Helvetica, Arial, sans-serif' } Need help or have a question?
 
 %p{ style: 'margin:1em 0;font: 16px Helvetica, Arial, sans-serif' } If you have a question or need any help with the online visits service, please #{link_to('contact us', new_feedback_url)}.
+
+%small
```
Comment (matharden) [inline app/views/mailers_common/_booking_confirmation.html.haml]: This should be in a <p> really.

Comment (jasiek) [inline app/views/mailers_common/_booking_confirmation.html.haml]: In a what?

File: app/views/mailers_common/_booking_rejection.html.haml [hunk 3]
```diff
@@ -76,3 +76,7 @@
   %a{ :href => prison_url(@visit) } prison website
   or call the prison on
   %strong #{prison_phone}.
+
+%small
```
Comment (matharden) [inline app/views/mailers_common/_booking_rejection.html.haml]: P element around this also.

File: app/views/prison_mailer/booking_request_email.html.haml [hunk 4]
```diff
@@ -52,3 +52,7 @@
 
 %div{ style: 'word-wrap: break-word; border: thin solid gray; padding: 5px;' }
   = deferred_new_confirmation_url(state: @token)
+
+%small
```
Comment (matharden) [inline app/views/prison_mailer/booking_request_email.html.haml]: P element again.

File: app/views/visitor_mailer/booking_receipt_email.html.haml [hunk 5]
```diff
@@ -59,3 +59,7 @@
   Need help or want to make a complaint? If you have a question or need any help with the online visits service, please 
   = succeed '.' do
     %a{ :href => new_feedback_url } contact us
+
+%small
```
Comment (matharden) [inline app/views/visitor_mailer/booking_receipt_email.html.haml]: Guess what…

File: spec/support/matchers.rb [hunk 6]
```diff
@@ -16,3 +16,16 @@
     part.body.raw_source.include?(actual)
   end
 end
+
+RSpec::Matchers.define :match_in_text do |actual|
```
Comment (matharden) [inline spec/support/matchers.rb]: :+1:

**Pull request merged**